Makefile

```diff
@@ -124,12 +125,12 @@ test: $(DEV_REQUIREMENTS_TIMESTAMP)
 
 # Serve targets. Using these will run the application on your local machine. You can either serve with a wsgi front (like it would be within the container), or without.
 .PHONY: serve
-serve: $(REQUIREMENTS_TIMESTAMP)
-	ENV_FILE=.env.default FLASK_APP=service_launcher FLASK_DEBUG=1 $(FLASK_CMD) run --host=0.0.0.0 --port=$(HTTP_PORT)
+serve: clean_logs $(LOGS_DIR)
```

Comment:
$(REQUIREMENTS_TIMESTAMP) should be left in dependencies with the new `clean_logs` and `$(LOGS_DIR)` dependencies.